src/overlays/actors/ovl_En_Part/z_en_part.c

```diff
@@ -28,16 +30,123 @@ const ActorInit En_Part_InitVars = {
     (ActorFunc)EnPart_Draw,
 };
 
-#endif
+Vec3f D_80865900 = { 0.0f, 0.0f, 0.0f };
+
+static void (*D_8086590C[5])(EnPart*, PlayState*) = {
```

Comment:
Make a typedef for this rather than use the nasty array-of-functions syntax. They're actionFunctions, so just adapt one from another actor and put it in the header.